pkg/azuredisk/nodeserver_test.go

```diff
@@ -718,14 +718,14 @@ func TestNodeExpandVolume(t *testing.T) {
 
 	invalidPathErr := testutil.TestError{
 		DefaultError: status.Error(codes.NotFound, "failed to determine device path for volumePath [./test]: path \"./test\" does not exist"),
-		WindowsError: status.Errorf(codes.NotFound, "Forward to GetVolumeIDFromTargetPath failed, err=error getting the volume for the mount ./test, internal error error getting volume from mount. cmd: (Get-Item -Path ./test).Target, output: (Get-Item -Path ), error: path \"./test\" does not exist"),
+		WindowsError: status.Errorf(codes.NotFound, "Forward to GetVolumeIDFromTargetPath failed, err=error getting the volume for the mount .\\test, internal error error getting volume from mount. cmd: (Get-Item -Path .\\test).Target, output: , error: <nil>"),
```

Comment:
I see that you have fixed the fakes mounter for windows. Be aware of the `/` and `\` as they tend to create mismatch and failures.